orc8r/cloud/go/services/orchestrator/analytics/analytics.go

```diff
@@ -15,17 +15,16 @@ package analytics
 
 import (
 	"magma/orc8r/cloud/go/services/analytics/calculations"
-	orchestrator_service "magma/orc8r/cloud/go/services/orchestrator"
 
```

Comment:
This file is more like orc8r library code -- I don't think it belongs under `services/...`

Reply:
Yes. It is library code. I will move this to orc8r/lib